src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: dcb92ce6070892036783bd78f76e5cda</sub>